test/workflows/components/workflows.libsonnet

```diff
@@ -183,6 +185,8 @@
               },
             },
           ],  // volumes
+          // onExit specifies the template that should always run when the workflow completes.
+          onExit: "exit-handler",
```

Comment:
This isn't clear to me. What's going on here?

Reply:
The steps is for check result as above answer, that will always run even if some component traceback or errors happen. Followed other projects in kubeflow done, for kubeflow examples https://github.com/kubeflow/examples/blob/264f48dfe80ba73fe73631d37568960001c5a8da/test/workflows/components/workflows.libsonnet#L152